src/main/java/pl/jeremi/antirelog/CombatHandle.java

```diff
@@ -16,12 +16,12 @@
 @SuppressWarnings("FieldCanBeLocal")
 class CombatHandle {
     static boolean enableBar;
-    private int combatTimeLeft;
+    public int combatTimeLeft;
```

Comment:
It's not like I'm in any position to complain right now, but I wouldn't make this `public` if I were you. :upside_down_face:

Reply:
I'm new in here and I can fix this

Reply:
I fixed this mistake and changed some things in my fork